pkgs/development/python-modules/mididings/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, buildPythonPackage
+, stdenv
+, python
+, fetchFromGitHub
+, pkgs
+, libjack2
+, alsaLib
+, pkgconfig
+, glib
+, boost
+, python36Packages
+}:
+
+buildPythonPackage rec {
+  pname = "mididings";
+  version = "unstable-2015-11-17";
+
+
+  patches = [ ./0001-add-boost-suffix.patch ];
+
+  LIBRARY_PATH="${(pkgs.boost.override {enablePython = true; inherit python; })}/lib";
```

Comment:
What is the purpose of this?

Reply:
Yes, without it it gives a linker error: 
/nix/store/vcc4svb8gy29g4pam2zja6llkbcwsyiq-binutils-2.30/bin/ld: cannot find -lboost_python